hail/python/test/hail/vds/test_vds_functions.py

```diff
@@ -28,7 +28,7 @@ def test_lgt_to_gt():
 
 def test_lgt_to_gt_invalid():
     c1 = hl.call(1, 1)
-    c2 = hl.call(1, 1, phased=True)
+    hl.call(1, 1, phased=True)
```

Comment:
The `c2` variable is referenced in the below commented out assertion. Let's turn that comment into an actual test.